include/dd/ComputeTable.hpp

```diff
@@ -34,8 +35,9 @@ namespace dd {
         static constexpr std::size_t MASK = NBUCKET - 1;
 
         static std::size_t hash(const LeftOperandType& leftOperand, const RightOperandType& rightOperand) {
-            const auto h1   = std::hash<LeftOperandType>{}(leftOperand);
-            const auto h2   = std::hash<RightOperandType>{}(rightOperand);
+            const auto h1 = std::hash<LeftOperandType>{}(leftOperand);
+            const auto h2 = std::hash<RightOperandType>{}(rightOperand);
+            // todo improve the hashing scheme
```

Comment:
In preliminary tests, I found that the hash function could be better. Due to time constrictions, I did not do any comprehensive benchmarks but left this comment as a reminder to get back to it. I removed the comment.

Reply:
Rather than having a comment in the code, where it might easily get overlooked, consider creating an issue on GitHub with precisely that goal. That way, it is much easier to capture these "low hanging fruits" or implementation details to be evaluated and tweaked. We even have the "enhancement" label for these purposes.